e2e/components/Header.spec.js

```diff
@@ -0,0 +1,65 @@
+describe('NEM 2 Explorer Header', () => {
+    beforeEach(() => {
+        cy.visit('/')
+    })
+
+    it('check on Title', () => {
```

Comment:
please apply same text style as above messages: `contain correct logo and title given click on Title`